cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/common/MetadataClient.java

```diff
@@ -94,6 +94,13 @@ public synchronized ClusterAndGeneration refreshMetadata(long timeout) {
     return new ClusterAndGeneration(_metadata.fetch(), _metadataGeneration.get());
   }
 
+  /**
+   * Close the metadata client.
```

Comment:
Did we mean network client? Otherwise, if `_networkClient` is used as the metadata client, should we rename it to `_metadataClient`?